boards/adafruit/feather_esp32s2/Kconfig.defconfig

```diff
@@ -6,6 +6,9 @@ config ENTROPY_GENERATOR
 
 if BOARD_ADAFRUIT_FEATHER_TFT || BOARD_ADAFRUIT_FEATHER_TFT_REVERSE
 
+config BOARD_LATE_INIT_HOOK
+	default y
```

Comment:
this needs removing and going back to the way I mentioned. These need squashing also